src/dotnet/commands/dotnet-uninstall/tool/LocalizableStrings.resx

```diff
@@ -145,12 +145,12 @@
     <value>Failed to uninstall tool '{0}': {1}</value>
   </data>
   <data name="UninstallToolCommandNeedGlobalOrToolPath" xml:space="preserve">
-    <value>Need either global or tool-path provided.</value>
+    <value>Please specify either the global option (--global) or the tool path option (--tool-path).</value>
   </data>
   <data name="ToolPathDescription" xml:space="preserve">
-    <value>Location of shim to access tool</value>
+    <value>Location where the tool will be installed.</value>
```

Comment:
This should be `Location where the tool was previously installed.`

Reply:
Changed as suggested